### ISSUE TITLE
Simulator inputs are being set to touch data even if simulator:set... is not being called

### ISSUE THREAD

**Issue by singularcitrus** [nameouschangey/Stormworks_VSCodeExtension#100]
Below is the onLBSimulatorTick function and supporting functions:

```lua
-- Runs every tick just before onTick; allows you to simulate the inputs changing
    ---@param simulator Simulator Use simulator:<function>() to set inputs etc.
    ---@param ticks     number Number of ticks since simulator started
    function onLBSimulatorTick(simulator, ticks)

        for i = 1, 0, 32 do
            simulator:setInputBool(i, false)
            simulator:setInputNumber(i, 0)
        end

        if output._bools[31] then
            block_touch(simulator, ticks)
        else
            block_not_touch(simulator, ticks)
        end
    end;

    function block_touch(simulator, ticks)
        -- touchscreen defaults
        --local screenConnection = simulator:getTouchScreen(1)
        --simulator:setInputBool(1, screenConnection.isTouched)
        --simulator:setInputNumber(1, screenConnection.width)
        --simulator:setInputNumber(2, screenConnection.height)
        --simulator:setInputNumber(3, screenConnection.touchX)
        --simulator:setInputNumber(4, screenConnection.touchY)
        print("meep")
    end
    
    function block_not_touch(simulator, ticks)
        -- Simulate stuff...
    end
```

As you can see on every simulator tick I reset all inputs to 0 and false, and only run `block_touch` if the mc script has set the output 31 to true, also in the code everything touch related is commented out and there is a `print("meep")` for debugging (obviously). When I run the script using "F6" the "meep" is never printed (because the mc has not set output 31 to true), which indicates that the function is not running, but when I click around on the virtual screen the inputs are still being set to the touch data. I also have a `print(input.getNumber(1))` in my mc code, which always outputs 288.0, which is the width of my defined 9x5 screen. This indicates to me that it's not just a difference between what the c# sim is drawing and lua is actually doing. When I completely deleted the commented out code the simulator kept outputting the screen data on the input channel, which it obviously should not be doing.

Let me know if you need anything more from me.

**Comment by singularcitrus** [comment]
Ok, update. I have solved the issue, the problem was a lack of documentation. In `Simulator.lua` round about line 324, there is a section called "enable touchscreen by default", which registers handlers for touch screen inputs, so no matter what you do it will always update inputs based on touch data. This isn't documented anywhere.

I managed to solve my specific issue by adding the following just before `function onLBSimulatorTick(simulator, ticks)`
```lua
simulator.config.numberHandlers = {}
simulator.config.boolHandlers = {}
```

With that said I would like to request an update to the README on how to disable touch by default.
